core/task.go

```diff
@@ -29,8 +29,10 @@ import (
 	"github.com/intelsdi-x/snap/scheduler/wmap"
 )
 
+// TaskState integer type
 type TaskState int
 
+// const task states
 const (
 	TaskDisabled TaskState = iota - 1
```

Comment:
TaskDisabled is the state of a task when it has reached its consecutive failure limit.